feature_creation_scripts/transport_transects/flipOSNAP_renameArcticTransectsTag.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+  
+import shutil
+
+from geometric_features import GeometricFeatures, FeatureCollection
+from geometric_features.utils import write_feature_names_and_tags
+
+
+gf = GeometricFeatures(cacheLocation='../../geometric_data',
+                       remoteBranchOrTag='master')
+
+fcNew = FeatureCollection()
+
+# Flip OSNAP sections and rename their tag to 'arctic_transport_sections'
+fc = gf.read('ocean', 'transect', ['OSNAP section East'])
+properties = fc.features[0]['properties']
+properties['tags'] = 'arctic_transport_sections'
+geometry = fc.features[0]['geometry']
+coords = geometry['coordinates']
+geometry['coordinates'] = coords[::-1]
+fcNew = fc
+
+fc = gf.read('ocean', 'transect', ['OSNAP section West'])
+properties = fc.features[0]['properties']
+properties['tags'] = 'arctic_transport_sections'
+geometry = fc.features[0]['geometry']
+coords = geometry['coordinates']
+geometry['coordinates'] = coords[::-1]
```

Comment:
```suggestion
geometry['coordinates'] = [coords[0][::-1]]
```

Reply:
this change (and similarly below) is not necessary.